src/Block/Service/AbstractBlockService.php

```diff
@@ -63,7 +72,13 @@ public function __construct($name = null, EngineInterface $templating = null)
      */
     public function renderResponse($view, array $parameters = [], Response $response = null)
     {
-        return $this->getTemplating()->renderResponse($view, $parameters, $response);
+        if (null === $response) {
```

Comment:
this can probably be changed to `$resposne = $response ?? new Response();` but I am not sure that makes it better 😄

Reply:
Fixed. But, i searched for `?? new` string in master branches of all sonata branches. This will be for a first time, where `$obj = $obj ?? new className()` is used =)